digitaltwins/device/src/digital_twin_client.ts

```diff
@@ -407,15 +417,57 @@ export class DigitalTwinClient {
       updateMessage.properties.add(commandUpdateStatusCodeProperty, status.toString());
       updateMessage.properties.add(messageInterfaceInstanceProperty, interfaceInstanceName);
       updateMessage.contentType = 'application/json';
+      updateMessage.contentEncoding = 'utf-8';
       this._client.sendEvent(updateMessage, (updateError) => {
         return _callback(updateError);
       });
     }, commandCallback);
   }
+  //
+  // Another of several functions that create and return a new function.
+  // In this case the returned function is used to send an arbitrary set
+  // of telemetry k/v pairs for a Digital Twin interfaceInstance.
+  //
+  private _returnSendTelemetryMethod(interfaceInstanceName: string): InterfaceTelemetryPromise | InterfaceTelemetryCallback {
+    return (telemetry, callback) => this._sendImplodedTelemetry(interfaceInstanceName, telemetry, callback);
+  }
+
+  /**
+   * @method                        private _sendImplodedTelemetry
+   * @description                   Sends a telemetry message for a supplied interface.
+   * @param interfaceInstanceName   Name of the instance for this interface.
+   * @param telemetry               The object to be sent.
+   * @param sendCallback (optional) If present, the callback to be invoked on completion of the telemetry,
+   *                                otherwise a promise is returned.
+   */
+  private _sendImplodedTelemetry(interfaceInstanceName: string, telemetry: any, sendCallback: ErrorCallback): void;
+  private _sendImplodedTelemetry(interfaceInstanceName: string, telemetry: any): Promise<void>;
```

Comment:
>sendImplodedTelemetry [](start = 11, length = 21)

After the message is imploded isn't it just a _sendTelemetryMessage

Reply:
When I remove the old code I'll change this internal name to remove the imploded.
Good comment